src/components/calcite-input/calcite-input.scss

```diff
@@ -1,3 +1,7 @@
+:host {
+  @apply block;
```

Comment:
@eriklharper what display should this component have? Should it be "block"? Seems like its already block now but I'm not sure why.

Reply:
cc @macandcheese

Reply:
`inline-block` is what the native input does, so that's probably the more appropriate one to use for this.

Reply:
If I change this to `inline-block` the screener tests will change due to width differences. I will leave as now. If you think it should be changed I think a different issue is needed.

Reply:
I do agree that both of these should probably be inline-block like their native counterparts. However, I didn't want to break screener tests and user apps. So this would likely be a breaking change.